### PR TITLE
6074 Update wait time tool tips

### PR DESCRIPTION
**Pull request by argush3**
*Issue #:* /bcgov/entity#6074

*Description of changes:*

* Updated both wait time tooltips to be 270px wide, text left-justified and arrow left nudge adjustment
* Added dotted underline styling to wait time labels to the right of the wait time circles
* Updated tooltip text for both wait times

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/new-request/stats.vue [hunk 1]
```diff
@@ -123,6 +125,8 @@ export default class Stats extends Vue {
   padding-left: 8px;
   font-size: 12px;
   line-height: 18px;
+  text-decoration-line: underline;
+  text-decoration-style: dotted;
```
Comment (severinbeauvais) [inline src/components/new-request/stats.vue:129]: Use base class `dotted-underline` instead.

Also, should this use class `cursor-default`?

**Review by andrepestana-aot — COMMENTED** [review]
LGTM!

**Review by andrepestana-aot — APPROVED** [review]

File: src/components/new-request/stats.vue (hunk 1 above)
Comment (argush3) [reply src/components/new-request/stats.vue:129]: Not sure about your comment regarding `cursor-default`.  There is no mention of what the cursor needs to be in the designs from what I can see.

Comment (severinbeauvais) [reply src/components/new-request/stats.vue:129]: I think we don't handle the tooltip hover cursor consistently in our apps...
Anyway, in this app, we don't change the cursor for tooltips, so this is fine.

**Comment by severinbeauvais** [comment]
I can't remember why we get the testing-coverage check error above (but I think we've seen it before). Please follow up with Patrick Wei. Don't merge until that's fixed please.

**Review by severinbeauvais — APPROVED** [review]
The code looks good!

**Comment by argush3** [comment]
> I can't remember why we get the testing-coverage check error above (but I think we've seen it before). Please follow up with Patrick Wei. Don't merge until that's fixed please.

ok, will follow up with Patrick.

**Comment by argush3** [comment]
@severinbeauvais talked to Patrick and it seems to have been just a connectivity issue.  He re-ran it and things checked out.

**Pull request merged**